helm-chart/openj9-jitserver-chart/README.md

```diff
@@ -0,0 +1,228 @@
+```
+
+#### Scenario 2: Only some applications are being upgraded to a newer version of JDK.
+If some applications are upgraded to a newer OpenJ9 JDK version while others are still using the older version of OpenJ9 JDK, you need to keep the existing JITServer and deploy another JITServer instance that is compatible with newer version of JDK. Replace `REPLACE_IMAGE_TAG` with the desired image tag to be deployed as a JITServer instance. 
+```bash
+helm install --set image.tag="REPLACE_IMAGE_TAG" jitserver-release-latest openj9-jitserver-chart
+```
+When all applications completed upgrades, and the older versions of the application are retired, you are safe to delete the older version of JITServer.
+``` bash
+helm delete jitserver-release
+```
+
+### Removing the JITServer
+Avoid removing the JITServer instance before applications terminate. There is a risk of application JVM has insufficient memory since it assumes JITServer handles the JIT Compilations.
+Once all applications are retired, you can safely remove the JITServer instance. 
+``` bash
+helm delete jitserver-release
+```
+
+## Resources Required
+### System
+The following are recommended minimum values for CPU and memory resources at JITServer, based on the assumption of a single application connecting to the JITServer instance:
+* CPU Requested : 1 CPU
+* Memory Requested : 512 MB
+If multiple applications connect concurrently to one JITServer instance, multiply the above values accordingly. Please note that, if multiple applications connect to the JITServer in a staggered fashion (i.e. with some delay between applications), then you could lower the above requirements.
+
+### Storage
+* Currently no storage or volume needed.
+
+## Chart Details
+This chart installs one JITServer instance that is ready to serve applications. It contains the following two K8s objects:
+* One `deployment` running AdoptOpenJDK OpenJ9 image
+* One `service` that is JITServer technology endpoint
+
+## Prerequisites
+### Red Hat OpenShift SecurityContextConstraints Requirements
+This chart does not require SecurityContextConstraints to be bound to the target namespace prior to installation.
+
+### PodSecurityPolicy Requirements
+This chart does not require PodSecurityPolicy to be bound to the target namespace prior to installation.
+
+### PodDisruptionBudgets Requirement
+This chart does not require PodDisruptionBudgets to be bound to the target namespace prior to installation.
+
+## Documentation
+* OpenJ9 website: https://www.eclipse.org/openj9/index.html
+* OpenJ9 JITServer technology: https://www.eclipse.org/openj9/docs/jitserver/
+* OpenJ9 repository: https://github.com/eclipse/openj9
+* OpenJ9 docs: https://github.com/eclipse/openj9-docs
+
+## Limitations
+* Deploys on Linux on x86-64, Linux on Power and Linux on Z 64-bit only. In the future other platforms may be supported as well.
+* Supports Java 8 and Java 11. Other Java versions may be supported in future releases.
```

Comment:
Good to link to the website (or somewhere) that's kept up to date with this info

Reply:
Will change to the JITServer page on OpenJ9 doc website. Looks like this is regularly updated and contains sufficient information on the latest releases. 
https://www.eclipse.org/openj9/docs/jitserver/